wd_helper.au3

```diff
@@ -3478,19 +3478,22 @@ EndFunc   ;==>__WD_Base64Decode
 
 ; #INTERNAL_USE_ONLY# ===========================================================================================================
 ; Name ..........: __WD_ErrHnd
-; Description ...: Dummy error handler
-; Syntax ........: __WD_ErrHnd()
-; Parameters ....: None
+; Description ...: COM Error handler
+; Syntax ........: __WD_ErrHnd($oError)
+; Parameters ....: $oError              - Error object.
 ; Return values .: None
 ; Author ........: mLipok
 ; Modified ......:
 ; Remarks .......:
-; Related .......:
+; Related .......: __WD_UpdateExtractor, _WD_DownloadAsBinary
```

Comment:
IMO, these should be listed here just because they call this function

Reply:
Done: [related removal](https://github.com/Danp2/au3WebDriver/pull/513/commits/5f1ae13760f7615ed8e96698cffa9c4575931518)